setup.cfg

```diff
@@ -44,6 +44,12 @@ addopts = --doctest-rst
 console_scripts =
     run_dgfit = dgfit.run_dgfit:main
     plot_dgfit = dgfit.plotting.plot_dgfit:main
+    #plot_dustmodel = dgfit.plotting.plot_dustmodel:main
+    plot_dustgrains = dgfit.plotting.plot_dustgrains:main
+    plot_dustgrains_alav = dgfit.plotting.plot_dustgrains_alav:main
+    plot_effsize = dgfit.plotting.plot_effsize:main
+    plot_obsdata = dgfit.plotting.plot_obsdata:main
+    plot_samples = dgfit.plotting.plot_samples:main
```

Comment:
These are a bit generic of names.  By this I mean other packages could have the same names.  I would suggest changing all the "plot" names to "dgplot" to make it clear this is for the dgfit package.

This is not the case for the 1st two aliases 'run/plot_dgfit' as those are already really specific to this package.